src/scrape_up/linkedIn/linkedInspider.py

```diff
@@ -0,0 +1,145 @@
+        except:
+            item['location'] = summary_box.css('span.top-card__subline-item::text').get().strip()
+            if 'followers' in item['location'] or 'connections' in item['location']:
+                item['location'] = ''
+
+        item['followers'] = ''
+        item['connections'] = ''
+
+        for span_text in summary_box.css('span.top-card__subline-item::text').getall():
+            if 'followers' in span_text:
+                item['followers'] = span_text.replace(' followers', '').strip()
+            if 'connections' in span_text:
+                item['connections'] = span_text.replace(' connections', '').strip()
+
+
+        """
+            ABOUT SECTION
+        """
+        item['about'] = response.css('section.summary div.core-section-container__content p::text').get(default='')
+
+
+        """
+            EXPERIENCE SECTION
+        """
+        item['experience'] = []
+        experience_blocks = response.css('li.experience-item')
+        for block in experience_blocks:
+            experience = {}
+            ## organisation profile url
+            experience['organisation_profile'] = block.css('h4 a::attr(href)').get(default='').split('?')[0]
+                
+                
+            ## location
+            experience['location'] = block.css('p.experience-item__location::text').get(default='').strip()
+                
+                
+            ## description
+            try:
+                experience['description'] = block.css('p.show-more-less-text__text--more::text').get().strip()
+            except Exception as e:
+                print('experience --> description', e)
+                try:
+                    experience['description'] = block.css('p.show-more-less-text__text--less::text').get().strip()
+                except Exception as e:
+                    print('experience --> description', e)
+                    experience['description'] = ''
+                    
+            ## time range
+            try:
+                date_ranges = block.css('span.date-range time::text').getall()
+                if len(date_ranges) == 2:
+                    experience['start_time'] = date_ranges[0]
+                    experience['end_time'] = date_ranges[1]
+                    experience['duration'] = block.css('span.date-range__duration::text').get()
+                elif len(date_ranges) == 1:
+                    experience['start_time'] = date_ranges[0]
+                    experience['end_time'] = 'present'
+                    experience['duration'] = block.css('span.date-range__duration::text').get()
+            except Exception as e:
+                print('experience --> time ranges', e)
+                experience['start_time'] = ''
+                experience['end_time'] = ''
+                experience['duration'] = ''
+            
+            item['experience'].append(experience)
+
+        
+        """
+            EDUCATION SECTION
+        """
```

Comment:
Hey, the code looks good and clean, but try to implement them as class methods.
For example, 
```python
user = Linkedin(username="nikhil25803")
user.experience()
```
This should return the user's experience detail as response. Else the code is clean and good. 
Just make these changes :))

Reply:
Sure @nikhil25803